src/postman/operation.ts

```diff
@@ -84,12 +104,11 @@ function findItem(collection: Collection, method: string, path: string): Item |
 }
 
 // this func ensures that both this securityScheme and the one in HttpService share exactly the sme keys
-function findPostmanSecurityScheme(auth: RequestAuth, collection: Collection) {
+function findPostmanSecurityScheme(auth: RequestAuth, securitySchemes: PostmanSecurityScheme[]) {
   const securityScheme = transformSecurityScheme(auth, () => '1');
   if (!securityScheme) return;
 
-  const allSecuritySchemes = transformSecuritySchemes(collection);
-  return allSecuritySchemes.find(ss => isPostmanSecuritySchemeEqual(ss, securityScheme));
+  return securitySchemes.find(ss => isPostmanSecuritySchemeEqual(ss, securityScheme));
```

Comment:
Ok so I am assuming you've decided to follow my advice of computing this somewhere else — I just need to find out where.

Reply:
All stays the same for single operation transformation. I just made this func more universal so it can be use sequentially. I think this comment may put some light on how `http-spec`s interface looks like: https://github.com/stoplightio/http-spec/pull/68/files#r381928417